tests/ctst/steps/azureArchive.ts

```diff
@@ -338,30 +339,81 @@ Then('manifest containing object {string} should contain {int} objects',
         assert.strictEqual(count, objectCount);
     });
 
-Then('blob for object {string} must be rehydrated', async function (this: Zenko, objectName: string) {
-    let found = false;
-    const {
-        tarName,
-    } = await findObjectPackAndManifest(
-        this,
-        this.getSaved<string>('bucketName'),
-        objectName || this.getSaved<string>('objectName'),
-    );
-    assert(tarName);
-    while (!found) {
-        found = await AzureHelper.blobExists(
+Then('blob for object {string} must be rehydrated',
+    async function (this: Zenko, objectName: string) {
+        let found = false;
+        const {
+            tarName,
+        } = await findObjectPackAndManifest(
+            this,
+            this.getSaved<string>('bucketName'),
+            objectName || this.getSaved<string>('objectName'),
+        );
+        assert(tarName);
+        while (!found) {
+            found = await AzureHelper.blobExists(
+                this.parameters.azureArchiveContainer,
+                `rehydrate/${tarName}`,
+                getAzureCreds(this),
+            );
+        }
+        await AzureHelper.sendBlobCreatedEventToQueue(
+            this.parameters.azureArchiveQueue,
             this.parameters.azureArchiveContainer,
             `rehydrate/${tarName}`,
             getAzureCreds(this),
         );
-    }
-    await AzureHelper.sendBlobCreatedEventToQueue(
-        this.parameters.azureArchiveQueue,
-        this.parameters.azureArchiveContainer,
-        `rehydrate/${tarName}`,
-        getAzureCreds(this),
-    );
-});
+    });
+
+Then('blob for object {string} fails to rehydrate',
+    async function (this: Zenko, objectName: string) {
```

Comment:
please introduce helper function (avoid duplicating this code!), and add comment to explain what/why this function does what it does (i.e. how it simulates a failure to rehydrate)